packages/workbox-broadcast-cache-update/test/sw/broadcast-update.js

```diff
@@ -1,22 +1,40 @@
-importScripts('/__test/mocha/sw-utils.js');
-importScripts('/__test/bundle/workbox-broadcast-cache-update');
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
 
-describe('Test of the broadcastUpdate function', function() {
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/* eslint-env mocha, browser */
+/* global expect */
+
+import {cacheUpdatedMessageType} from '../../src/lib/constants.js';
+import broadcastUpdate from '../../src/lib/broadcast-update.js';
+
+describe(`Test of the broadcastUpdate function`, () => {
   const channelName = 'test-channel';
   const channel = new BroadcastChannel(channelName);
   const cacheName = 'test-cache';
   const url = 'https://example.com';
   const source = 'test-source';
 
-  it(`should throw when broadcastUpdate() is called without any parameters`, function() {
-    expect(() => workbox.broadcastCacheUpdate.broadcastUpdate()).to.throw();
+  it(`should throw when broadcastUpdate() is called without any parameters`, () => {
+    expect(() => broadcastUpdate()).to.throw(TypeError);
```

Comment:
The original code predated some of the changes to `assert.js`.

I think the best practice nowadays would be

```js
expect(() => broadcastUpdate()).to.throw('assertion-failed');
```

That will work with both the `prod` and `dev` bundles, as it checks the `name` property of the thrown exception, and in both cases that will be set to `'assertion-failed'`.